test/e2e/multinamespace_test.go

```diff
@@ -97,7 +97,7 @@ var _ = Describe("Users can use clusterwide configuration with limitation to wat
 				actions.PrepareUsersConfigurations(&listData[i])
 			}
 			deploy.MultiNamespaceOperator(&listData[0], watchedNamespace)
-			kubecli.CreateApiKeySecret(config.DefaultOperatorGlobalKey, config.DefaultOperatorNS)
+			kubecli.CreateDefaultSecret(listData[0].Context, listData[0].K8SClient, config.DefaultOperatorGlobalKey, config.DefaultOperatorNS)
```

Comment:
Minor thing: can assign `dataItem := listData[0]`, check if it's not empty and then use it everywhere)

Reply:
Done